ligo/stablecoin/fa1.2/spec.ligo

```diff
@@ -143,6 +143,8 @@ type get_default_expiry_param is michelson_pair_right_comb(record
 ; callback : contract(seconds)
 end)
 
+const permit_expiry_limit: nat = 31557600000n; // A thousand years
```

Comment:
Can we avoid having this constant in more than one place?

Reply:
Changed